docs/misc/19_enterprise_onboarding/index.mdx

```diff
@@ -0,0 +1,370 @@
+# Enterprise onboarding
+
+import DocCard from '@site/src/components/DocCard';
+
+Welcome to [Windmill Enterprise Edition](/pricing). This guide provides essential setup information and best practices for new enterprise customers.
+
+## Getting started
+
+### Docker images
```

Comment:
Container Images

and on helm chart it's done automatically when you use :
```
enterprise: 
  enabled: true 
```
in the values.yaml which is what enterprise will mostly do. Should also mention that k8s is recommended for serious deployment